projects/ngrx-auto-entity/src/lib/selectors/tracking.selectors.ts

```diff
@@ -1,5 +1,15 @@
 import { IEntityTracking } from '../util/entity-state';
 
+// prettier-ignore
+export const mapToHasBeenLoaded =
+  (tracking: IEntityTracking): boolean =>
+    tracking?.isLoading != null;
```

Comment:
I think these are swapped. Wouldn't `hasBeenLoaded` be `loadedAt != null`

Reply:
Ah, good catch. Adding now.

Reply:
Ok, I've swapped these. I was thinking about it differently, but I think it does make more sense with them swapped from how they were.